src/metabolic_ninja/resources.py

```diff
@@ -26,11 +26,24 @@
 
 class PredictionJobsResource(MethodResource):
     @use_kwargs(PredictionJobRequestSchema)
-    def post(self, model_name, product_name, max_predictions):
+    def post(self, bigg, kegg, rhea, model_name, product_name, max_predictions,
+             product, project_id, model, species):
         result = predict.delay(model_name, product_name, max_predictions)
         return {
             'id': result.id,
             'state': result.state,
+            'configuration': {
```

Comment:
This is just echoing back exactly what was posted, that seems a bit unnecessary - the client already knows this information.